graphql_api/types/segment_comparison/segment_comparison.graphql

```diff
@@ -1,5 +1,5 @@
 type SegmentComparison {
-  header: String
+  header: String!
```

Comment:
Could you please adjust the return type of this resolver to reflect this? Still says optional 👌